### ISSUE TITLE
`pod update`ing a specific messes up other pods referenced by tag or commit (0.36)

### ISSUE THREAD

**Issue by ayanonagon** [CocoaPods/CocoaPods#3292]
This only happens on `0.36` (works fine on `0.35`).

I tried to `pod update some-specific-pod`, then that [incorrectly] updated the references for all other dependencies that are referenced by git repo + (commit|tag). It seems to have made them point to really old versions. Let me know what other info I can give!

:octocat: 

P.S. Congrats on the Swift release! :confetti_ball:

**Comment by segiddins** [comment]
Can you please share the diff in your lockfile that the update caused, as well as your Podfile? Thanks Ayaka!

**Comment by FredericJacobs** [comment]
Hi,
I had to downgrade to 0.35 too because on 0.36 running `pod install` doesn't take into account the `:git`, `commit` and `:branch` parameter being passed.

These are some entries of my `Podfile` that are being installed but without respecting the extra parameters.


```
pod 'SocketRocket',               :git => 'https://github.com/FredericJacobs/SocketRocket.git', :commit => '73d8a19'
pod 'JSQMessagesViewController',  :git => 'https://github.com/WhisperSystems/JSQMessagesViewController', :branch => 'JSignalQ'
pod 'APDropDownNavToolbar',       :git => 'https://github.com/corbett/APDropDownNavToolbar.git', :branch => 'master'
pod 'EZAudio',                    :git => 'https://github.com/jackflips/EZAudio', :commit => '1620f19'
```

**Comment by segiddins** [comment]
@ayanonagon just following up on my earlier request for more info?

**Comment by ayanonagon** [comment]
Sorry for the delay, here comes a lot of output:

`Podfile` (the line I changed before running `pod update venmo-ios-images` was `pod 'venmo-ios-colors', :git => "git@github.com:venmo/venmo-ios-colors.git", :tag => 'v0.6.0'` (from `v1.4.1`):
```ruby
source 'https://github.com/CocoaPods/Specs.git'
source 'https://github.braintreeps.com/venmo/CocoaPods.git'

platform :ios, '7.0'

workspace 'Venmo'
xcodeproj 'VenmoIOS/Venmo'

inhibit_all_warnings!

target 'Venmo' do
  # Venmo Pods
  pod 'VENCalculatorInputView', :head
  pod 'VENCoreExtensions', :git => "git@github.braintreeps.com:venmo/VENCoreExtensions.git"
  pod 'VENExperimentsManager', '~> 1.0'
  pod 'VENNearbyAnimatingView', :podspec => 'PodSpecs/VENNearbyAnimatingView.podspec'
  pod 'VENNearbyManager', :podspec => 'PodSpecs/VENNearbyManager.podspec'
  pod 'VENSeparatorView', '~> 1.0.1'
  pod 'VENTokenField', '~> 2.3'
  pod 'VENTouchLock', '~> 1.0'
  pod 'Venmo-iOS-SDK', :podspec => 'PodSpecs/Venmo-iOS-SDK.podspec'
  pod 'VenmoClient', '~> 1.1.0'
  pod 'venmo-ios-images', :git => "git@github.braintreeps.com:venmo/venmo-ios-images.git", :tag => 'v1.6.0'
  pod 'venmo-ios-colors', :git => "git@github.com:venmo/venmo-ios-colors.git", :tag => 'v0.4.0'
  pod 'venmo-ios-fonts', :git => "git@github.com:venmo/venmo-ios-fonts.git", :tag => 'v0.3.0'

  # Podspec overrides
  pod 'BZGFormViewController', :podspec => 'PodSpecs/BZGFormViewController.podspec'
  pod 'CMPopTipView', :podspec => 'PodSpecs/CMPopTipView.podspec'

  # Other OSS Pods
  pod 'AFNetworking', '~> 2.5.0'
  pod '1PasswordExtension', '~> 1.0'
  pod 'BZGMailgunEmailValidation', '~> 1.1.1'
  pod 'BlocksKit', '~> 2.0'
  pod 'Bolts', '~> 1.1.3'
  pod 'Braintree/API', '~> 3.4.1'
  pod 'Braintree/UI', '~> 3.4.1'
  pod 'DAZABTest', '~> 1.0'
  pod 'DSXActivityIndicator', '~> 0.0.1'
  pod 'Facebook-iOS-SDK', '~> 3.22'
  pod 'FLEX', '~> 2.0.0'
  pod 'FXBlurView', '~> 1.6.3'
  pod 'FlurrySDK', '~> 5.0.0'
  pod 'FrameAccessor', '~> 1.1.0'
  pod 'GBDeviceInfo', '~> 2.2.8'
  pod 'Godzippa', '~> 1.0.0'
  pod 'HockeySDK', '~> 3.6'
  pod 'JTSImageViewController', '~> 1.4'
  pod 'KGStatusBar', '~> 1.0'
  pod 'MCDateExtensions', '~> 1.0.0'
  pod 'MMDrawerController', '~> 0.5.3'
  pod 'MZFormSheetController', '~> 2.3.5'
  pod 'Mixpanel', '~> 2.3.6'
  pod 'NSURL+QueryDictionary', '~> 1.0.3'
  pod 'ObjectiveSugar', '~> 1.1.0'
  pod 'pop', '~> 1.0.7'
  pod 'PSTAlertController', '~> 1.1.0'
  pod 'ReactiveCocoa', '~> 2.0'
  pod 'SDWebImage', '~> 3.7'
  pod 'SSKeychain', '~> 1.2.2'
  pod 'SVProgressHUD', :git => 'https://github.com/TransitApp/SVProgressHUD.git', :commit => 'c9a2904b5bd57786b05b61b60695764bff43538b'
  pod 'TOWebViewController', '~> 2.0'
  pod 'EDColor', '~> 1.0'

  pod 'TTTAttributedLabel', '~> 1.13.0'
  pod 'UAModalPanel', '~> 1.1.1'
  pod 'Underscore.m', '~> 0.2.0'
  pod 'iRate', '~> 1.9.3'
  pod 'libextobjc', '~> 0.4.1'
  pod 'Mantle', '~> 1.5'

  # Per-config Pods
  pod 'Lookback', '~> 0.9', :configurations => ['Debug']
  pod 'LookbackSafe', '~> 0.9', :configurations => ['Release']
end

target 'Tests' do
  link_with %w{
    Regression\ Tests
    Smoke\ Tests
    Unit\ Tests
    Staging\ Verification
  }
  pod 'KIF', '~> 3.1'
  pod 'Specta', :git => "https://github.com/specta/specta", :tag => 'v0.3.0.beta1'
  pod 'Expecta', '~> 0.3.1'
  pod 'OCMock', '~> 2.2.2'
  pod 'Nocilla', :head
end
```

`git diff Podfile.lock`:

```
diff --git a/Podfile.lock b/Podfile.lock
index 0fd4709..062b107 100644
--- a/Podfile.lock
+++ b/Podfile.lock
@@ -152,7 +152,7 @@ PODS:
   - TTTAttributedLabel (1.13.0)
   - UAModalPanel (1.1.1)
   - Underscore.m (0.2.1)
-  - VENCalculatorInputView (HEAD based on 1.4.0)
+  - VENCalculatorInputView (HEAD based on 1.5.0)
   - VENCore (3.1.0):
     - CMDQueryStringSerialization (~> 0.2.0)
   - VENCoreExtensions (2.0.1):
@@ -216,7 +216,8 @@ DEPENDENCIES:
   - SDWebImage (~> 3.7)
   - Specta (from `https://github.com/specta/specta`, tag `v0.3.0.beta1`)
   - SSKeychain (~> 1.2.2)
-  - SVProgressHUD (from `https://github.com/TransitApp/SVProgressHUD.git`, commit `c9a2904b5bd57786b05b61b60695764bff43538b`)
+  - SVProgressHUD (from `https://github.com/TransitApp/SVProgressHUD.git`, commit
+    `c9a2904b5bd57786b05b61b60695764bff43538b`)
   - TOWebViewController (~> 2.0)
   - TTTAttributedLabel (~> 1.13.0)
   - UAModalPanel (~> 1.1.1)
@@ -226,7 +227,8 @@ DEPENDENCIES:
   - VENExperimentsManager (~> 1.0)
   - venmo-ios-colors (from `git@github.com:venmo/venmo-ios-colors.git`, tag `v0.4.0`)
   - venmo-ios-fonts (from `git@github.com:venmo/venmo-ios-fonts.git`, tag `v0.3.0`)
-  - venmo-ios-images (from `git@github.braintreeps.com:venmo/venmo-ios-images.git`, tag `v1.4.1`)
+  - venmo-ios-images (from `git@github.braintreeps.com:venmo/venmo-ios-images.git`,
+    tag `v1.6.0`)
   - Venmo-iOS-SDK (from `PodSpecs/Venmo-iOS-SDK.podspec`)
   - VenmoClient (~> 1.1.0)
   - VENNearbyAnimatingView (from `PodSpecs/VENNearbyAnimatingView.podspec`)
@@ -256,7 +258,7 @@ EXTERNAL SOURCES:
     :tag: v0.3.0
   venmo-ios-images:
     :git: git@github.braintreeps.com:venmo/venmo-ios-images.git
-    :tag: v1.4.1
+    :tag: v1.6.0
   Venmo-iOS-SDK:
     :podspec: PodSpecs/Venmo-iOS-SDK.podspec
   VENNearbyAnimatingView:
@@ -282,72 +284,72 @@ CHECKOUT OPTIONS:
     :tag: v0.3.0
   venmo-ios-images:
     :git: git@github.braintreeps.com:venmo/venmo-ios-images.git
-    :tag: v1.4.1
+    :tag: v1.6.0
 
 SPEC CHECKSUMS:
-  1PasswordExtension: 37d1488cf049b8bc0811beb7b52a19f8f803f7c8
-  AFNetworking: 0f54cb5d16ce38c1b76948faffb8d5fb705021c7
-  BlocksKit: 4439e7f30a9f90743462ca63545a15c1f4304cef
-  Bolts: 3db0a875235e0c9042b013af13af0f00f79b102c
-  Braintree: 114c4c56e686ad296ac45bbc63550682c7b862c8
-  BZGFormViewController: 3074a6e9281723a67a7aa55b0a6879cb1813f513
-  BZGMailgunEmailValidation: 134975d405312a9e647785e9778861301516c333
-  CMDQueryStringSerialization: e53570ccbce6004acba30b1c1210e254644cdb92
-  CMPopTipView: 0dc0d7ee14c4ad13b151c2d9a61b19f2c514b4ed
-  DAZABTest: 68f482f27733272d8e56bceb87561517af45ff43
-  DSXActivityIndicator: 5994788fe8987f36c2d59f8995a3c3dddaf21e79
-  EDColor: 1638bf113678826a05855cff095fcec1d887dbb6
-  Expecta: 03aabd0a89d8dea843baecb19a7fd7466a69a31d
-  Facebook-iOS-SDK: 0eda8ff7120aada0a9c5f98552f7625da6c3440a
-  FLEX: d6818fd89f0b6c7c412549a5580e87d5cbaef885
-  FlurrySDK: 1fa28436e86e533e51ab1164c43d8f3b5f8ff99f
-  FrameAccessor: 1900268ec0c4872ce0948baaf71f995ea6d0f28c
-  FXBlurView: abe1046285813c4f6e9008abbc324fc5ab32af90
-  GBDeviceInfo: 59f94d06749cf6220d8b7492a0d1b2b310bb0ed2
-  Godzippa: 1aa13727d7abc32ef870c65051a41ce06cae2ad8
-  HockeySDK: bfd1f5ac75938b07499c4ac12932244b72a2e70b
+  1PasswordExtension: 26d590105f83349cd4fe362921534f8244b8e67b
+  AFNetworking: 96ac9bf3eda33582701cb1fcc5b896aa1e20311e
+  BlocksKit: 7f422b971407001178d181a43b99014ea2591fe6
+  Bolts: e431c88fd02115d4a34016c2100bbfa4d2d5b146
+  Braintree: d4aaf1e2c44d22e6457c6baa076a8b34f75441ab
+  BZGFormViewController: 1124746c943d8c47b9ed17c9f4d17b77efcd2692
+  BZGMailgunEmailValidation: 9951fc8e2a9435d2fa82ac04cb9e8c6cca7af3fc
+  CMDQueryStringSerialization: 15f9004212bbc72533bb533f63df79d90a00a7d8
+  CMPopTipView: 7683e991d98a5f9d5f913afc7ffb7052e9bb669e
+  DAZABTest: 40415a691be80877ee20ea66d33eef64995d7761
+  DSXActivityIndicator: a0bd2ec582655e0219d6d1fce8857bf8b72fd358
+  EDColor: c83f9a61f9f9b3c23d541f1feb561558e29cb088
+  Expecta: a354d4633409dd9fe8c4f5ff5130426adbe31628
+  Facebook-iOS-SDK: 141cba3ba6c484380eb38c573110af9e4fbf8962
+  FLEX: 9b3d0800d0c06887a00ad8d2b514fb69256ebd1b
+  FlurrySDK: a93f46ec37a762d11a4d6c5e5f06d0f12e995cd4
+  FrameAccessor: dbefce2d8973deafdefb7124aa767aa37dd860eb
+  FXBlurView: c6d23f3d35af2c6282296a2930f61c6e2c788d01
+  GBDeviceInfo: 817caabdef3fb4807fe45eda4aa8c858172f273c
+  Godzippa: a29dc387c5f83409ddd1a85e8c3f4132549705c7
+  HockeySDK: 51ade1aff04463137e852797d2eb4dfb2511d6a9
   iRate: 34b63a5e9dabcd26a5cf573d872ebb366802d63d
-  JTSImageViewController: 9fe6402194b7679f0e28d0c1002d6164301fc34a
-  KGStatusBar: b542c67ae300d91ebc22172e22c0389431a915e7
-  KIF: 068074b24dd455025e7f2a488ba4fb07b9ce047b
+  JTSImageViewController: 6a367b144e257db02cdfacdba78d684377d3777b
+  KGStatusBar: 084b81bc95975bcdf587bc410c8243b048e6b1e9
+  KIF: e08915d78a220480599837e9a760deee64453d76
   libextobjc: d6146fce3ebea986b7f75fcb4c967639200b88b0
   libPhoneNumber-iOS: 98fc07d70c8fdb5e6a8e3442c37e97353065c20e
-  Lookback: 6bfb2bbd2f106677d1615db64e91a47fd4c9bf93
-  LookbackSafe: b96af7166fe5ca6d81f70004ff4c4b0cdfd42290
-  Mantle: 8d84cacd6c2a69ff6fbce985a2b51298a5495de3
-  MCDateExtensions: f37ffd43f216cd075b7dc3bbc213dd680e5f74ea
-  Mixpanel: 9aba38257fc4d2777ab5db1afc55e22bdfd57a4a
-  MMDrawerController: c3ab7a318ddc7e2bcd133139c3161af08c6e1197
-  MZAppearance: c88502c0c2345120630c34e09d5ff09e2969ba30
-  MZFormSheetController: a7dce224ce8bd9015abeabca8cb6cde0d49d3fe3
-  Nocilla: 4d713c56e7357fd0cac0363854d4a58021ad3f41
-  NSURL+QueryDictionary: 1ad81be60c10e3f4d80c5c4beb7125de96aaec5e
-  ObjectiveSugar: f48793f5902c77cfc02a4ce46d2e8ecddcad4dc2
-  OCMock: 6db79185520e24f9f299548f2b8b07e41d881bd5
+  Lookback: 5855c41118f80f98f70cca5ea3219a8ab84676fe
+  LookbackSafe: 7c6f24c9aaf0ac2e4ce763269fd3a181dc308abb
+  Mantle: 7dddb082367d3e13bd244636d56838481c4552fe
+  MCDateExtensions: 469f38d0ed344653684881d53240f6ae21939d07
+  Mixpanel: dc56111f1b327b38e0ae5a805f33f5d0c9e7f7f2
+  MMDrawerController: 7a4cf8e25596bf2434e03ce50d22a8608ddee6a1
+  MZAppearance: c2937318234fd782e900fb178942278881ac5490
+  MZFormSheetController: e3c0accfab12ed1f9ddf2c61bb8c0abfc6b3f87b
+  Nocilla: d7d96e8a11e363f0d7c976d48b320c43d61fb471
+  NSURL+QueryDictionary: 06f93589c2948128321ea562592ed6bf1c84d070
+  ObjectiveSugar: a6a25f23d657c19df0a0b972466d5b5ca9f5295c
+  OCMock: a6a7dc0e3997fb9f35d99f72528698ebf60d64f2
   pop: 2f14a1ea61339767af9e66741b418c831b3844df
-  PSTAlertController: 69d94bd6b4954681700f43ab32e99727364e4ef7
-  ReactiveCocoa: dca75777b9d18990c8714a3a6f2149f29b4d1462
-  SDWebImage: 116e88633b5b416ea0ca4b334a4ac59cf72dd38d
-  Specta: 63e8ac8e07db6675c7096d37f77a266a1be4c02d
-  SSKeychain: cc48bd3ad24fcd9125adb9e0d23dd50b8bbd08b9
-  SVProgressHUD: 5e10939a296debf5fe5a5895eda9539b8b2114f1
-  TOWebViewController: d3eee5342e5f8508cc87107c96544f5dff01b802
-  TTTAttributedLabel: a847b59a75c92d59f4eabc9cc1745704cff2dc36
-  UAModalPanel: 9363c43a927fa8374a73366c8524b6ab22368b66
-  Underscore.m: f99c166a9fdccfb5069085b6b4263c04b817fc9f
-  VENCalculatorInputView: 9b413ef586949306b7aaad20693a463681afd678
-  VENCore: aaca1e3d666caab31f277492585bec9dcf1c0638
-  VENCoreExtensions: 3f430bdf55487a5caa12a110be89f72ab4a20343
-  VENExperimentsManager: f31796e5157582f2da31e000d0e02e499ae91471
-  venmo-ios-colors: 9015b53fe5bdef604dafaf0f00695471925d152d
-  venmo-ios-fonts: 2a9f4defc6d41b9c49d13e52417fa614fba67c86
-  venmo-ios-images: 432a0c9361e0d915b15f82267d7ec4eee9318054
-  Venmo-iOS-SDK: 8c583905632fa44f30ece593f1f5580e35d72b61
+  PSTAlertController: cb21afcfa76c3229feec7db044afd4f60dab5892
+  ReactiveCocoa: 334468a83b2c5f669f7cfb9a084415ee23219b4f
+  SDWebImage: ed3095af2ff88b436426037444979b917f6c5575
+  Specta: d2158a483ad179e5f8d3a29f75ff4fc0ff6fab16
+  SSKeychain: 88767e903ee8d274ed380e364d96b7a101235286
+  SVProgressHUD: 46088807596a795cbcb4affd92cf3f13b6ab3dda
+  TOWebViewController: 2cfd0032d781cabd88a0995fbca9692c11f0797e
+  TTTAttributedLabel: 8aeaf67481bb38970711e2641211cba912bd5239
+  UAModalPanel: f53b27c1c2704b49335c14ccfab1b427167cb8b0
+  Underscore.m: b86ca364d60509a39eb9c7886443c063b793aaba
+  VENCalculatorInputView: 000af0f9b878fe44895d85c0fedca075adfe81c5
+  VENCore: 17605707033535e5c0931d2853024fcee9b68b9e
+  VENCoreExtensions: c004443c180c8a6977cdd7c2e94419fdbcb2a2ee
+  VENExperimentsManager: 521a63415b515723a384a71c8ada61996198ce72
+  venmo-ios-colors: 05458d2a61bac604695738199aeb846b5cc120ce
+  venmo-ios-fonts: 5fac9666ad180b90f996ba602152ea92ec0d9ac0
+  venmo-ios-images: 03ced0c93db859dbcb76f1789d0ec0c6bc659387
+  Venmo-iOS-SDK: 8bcceeed904413acdf4fef049793693a097f0123
   VenmoClient: 24e6f4d1dc74a60b2dc9c800be542c9d8f8a5f24
-  VENNearbyAnimatingView: 3b1f4241519aeefc80c2aba55a6def27f833e922
-  VENNearbyManager: 333214618ad1dd76471a219a34d76fa53f8fcde8
-  VENSeparatorView: 402a48d9ba37eecad30bc806dab17e76f7a0818b
-  VENTokenField: fb3cee03bab62e8d7d501f987ac5a7316ccbd8fe
-  VENTouchLock: d6d65a8f502d46f8707e9cc9d0065456f248555f
+  VENNearbyAnimatingView: 18d052d8b5dd2e2e17f868066f46e15bd9fc0ef7
+  VENNearbyManager: 19a2dcd2db941f594cebbd8a2ada06ab154800e1
+  VENSeparatorView: b4c50599d11b9af22794e6cc4a5eb3374451b09a
+  VENTokenField: ec847952d7a6d4ad65cb8ba2fc8141ab26a041fa
+  VENTouchLock: d3b4f07229ca352a49057ed55bffe894d6701357
 
-COCOAPODS: 0.35.0
+COCOAPODS: 0.36.1
```

It seems like the Pods that we refer to by `:git` and `:podspec` got rolled back to a really early commit or something. Not sure how helpful the output is because some of our pods our internal. `VENCoreExtensions` (one of our internal pods) is most notably messed up.

Also part of `git status` shows that a bunch of code in non-`venmo-ios-images` got un-updated to older versions:
```
	modified:   Pods/VENCalculatorInputView/README.md
	modified:   Pods/VENCalculatorInputView/VENCalculatorInputView/VENCalculatorInputTextField.m
	modified:   Pods/VENCalculatorInputView/VENCalculatorInputView/VENMoneyCalculator.m
	modified:   Pods/VENCoreExtensions/VENCoreExtensions/Networking/VENEnvironment.m
	modified:   Pods/VENCoreExtensions/VENCoreExtensions/Networking/VENHTTPInternal.m
	deleted:    Pods/VENCoreExtensions/VENCoreExtensions/Notifications/VENNotification.h
	deleted:    Pods/VENCoreExtensions/VENCoreExtensions/Notifications/VENNotification.m
	deleted:    Pods/VENCoreExtensions/VENCoreExtensions/Transactions/VENTransaction+VENCoreExtensions.h
	deleted:    Pods/VENCoreExtensions/VENCoreExtensions/Transactions/VENTransaction+VENCoreExtensions.m
	modified:   Pods/VENCoreExtensions/VENCoreExtensions/Users/VENUser+VENCoreExtensions.h
	modified:   Pods/VENCoreExtensions/VENCoreExtensions/Users/VENUser+VENCoreExtensions.m
```

Let me know if I can give more info, thanks! :octocat:

**Comment by kylef** [comment]
Seems the pods that we're incorrectly updated are ones that used `:head` and the underlying spec in the specs repository was updated since you last `pod updated`. Sounds like `:head` specs are not correctly locked in `pod update <another pod>`.

---

Most of the spec checksum changes are due to most the specs in the master repo changing: https://github.com/CocoaPods/Specs/pull/13027.

This seems odd, but the diff shows some changes purely just word wrapping:

```diff
-  - SVProgressHUD (from `https://github.com/TransitApp/SVProgressHUD.git`, commit `c9a2904b5bd57786b05b61b60695764bff43538b`)
+  - SVProgressHUD (from `https://github.com/TransitApp/SVProgressHUD.git`, commit
+    `c9a2904b5bd57786b05b61b60695764bff43538b`)
```

**Comment by segiddins** [comment]
Yeah, the bit about `HEAD` pods is kinda behaves as intended. `HEAD` basically means "find the latest published podspec for this pod, and download it, ignoring the commit/tag". Since the version its based on changed, it will need to be re-downloaded.

**Comment by ayanonagon** [comment]
Thanks guys! What about the non `HEAD` ones? It looks like the lock file is properly maintaining the version, but it’s still being changed, based on `git status` and also by the fact that the project doesn’t build after `pod update venmo-ios-images` due to missing files from pods that were not supposed to be updated. For example, `VENCoreExtensions`. Not sure what is going on. :scream:

:octocat:

**Comment by segiddins** [comment]
What's the verbose output that happens when running that `pod update` command? It should say what git actions are being taken

**Comment by ayanonagon** [comment]
#3299 seems to have a much simpler steps-to-reproduce: https://github.com/CocoaPods/CocoaPods/issues/3299#issuecomment-88122671

**Comment by ayanonagon** [comment]
I just updated our `Podfile` to never use `HEAD` to reference pods, but pods referenced by `:commit` and `:tag` definitely still get screwed up. For example,

```ruby
pod 'SVProgressHUD', :git => 'https://github.com/TransitApp/SVProgressHUD.git', :commit => 'c9a2904b5bd57786b05b61b60695764bff43538b'
```

Some verbose output:

```
Creating possibility state for SVProgressHUD (from `https://github.com/TransitApp/SVProgressHUD.git`, commit `c9a2904b5bd57786b05b61b60695764bff43538b`) (1 remaining)
                                                                                                                                                                              Attempting to activate SVProgressHUD (1.0)
                                                                                                                                                                              Activated SVProgressHUD at SVProgressHUD (1.0)
```

```
-> Installing SVProgressHUD 1.0 (was 1.0)
 > Git download
 > Git download
     $ /usr/bin/git clone https://github.com/samvermette/SVProgressHUD.git /Users/ayakanonaka/venmo-ios/Pods/SVProgressHUD --single-branch --depth 1 --branch 1.0
     Cloning into '/Users/ayakanonaka/venmo-ios/Pods/SVProgressHUD'...
     Note: checking out '9cb74e5a719ce4c55ddfeab6e58407e790dd8341'.
     
     You are in 'detached HEAD' state. You can look around, make experimental
     changes and commit them, and you can discard any commits you make in this
     state without impacting any branches by performing another checkout.
     
     If you want to create a new branch to retain commits you create, you may
     do so (now or later) by using -b with the checkout command again. Example:
     
       git checkout -b new_branch_name
```

`c9a2904b5bd57786b05b61b60695764bff43538b` vs `9cb74e5a719ce4c55ddfeab6e58407e790dd8341`, seems like the commit tag is being ignored, 

Let me know what other info would help!

**Comment by segiddins** [comment]
OK, so it's definitely downloading the wrong branch. I need to look into this more in-depth later.

**Comment by ayanonagon** [comment]
Thank you @segiddins! :smile_cat:

**Comment by segiddins** [comment]
It actually should never have re-downloaded SVProgressHUD at all, it should've been using the pre-downloaded pod...

**Comment by segiddins** [comment]
This is fixed in 0.36.2.
